tests/installation/partitioning_filesystem.pm

```diff
@@ -23,16 +24,27 @@ sub run {
     assert_and_click 'edit-proposal-settings';
 
     if (get_var('PARTITIONING_WARNINGS')) {
-        assert_screen 'proposal-will-overwrite-manual-changes';
-        send_key 'alt-y';
+        if (is_storage_ng && check_screen 'partition-scheme') {
```

Comment:
that's not what I meant when I said that I consider the `check_screen …, 0` unsafe after clicking the previous button. I don't want to waste these 30s. As you know there are some people that just look at an openQA job run that already takes 2h wasting time so don't make it worse please

Reply:
The difference is that partition-scheme will match, and if not, test will be broken again as product has been changed. If it's not storage_ng we don't wait 30 seconds anyway. So we don't waste 30 seconds here.